### PR TITLE
Added destroy flag and info to namespace and preview destroy

### PR DESCRIPTION
**Pull request by teresaromero**
Release https://github.com/okteto/okteto/releases/tag/2.11.0 introduced the flag `--all` into the command `okteto destroy` in order to delete all the resources that are present at the current namespace.

Also, `okteto preview destroy` and `okteto namespace destroy` now execute the commands that are defined under the `destroy` section of the okteto manifest.

### CODE REVIEW

**Comment by rberrelleza** [comment]
@teresaromero in which version of Self-Hosted will this be available?

**Comment by teresaromero** [comment]
> @teresaromero in which version of Self-Hosted will this be available?

as it is at 2.11 i believe this will be at the next self-hosted release which would be 1.4 if i'm not mistaken? 
cc: @ifbyol

**Comment by ifbyol** [comment]
> > @teresaromero in which version of Self-Hosted will this be available?
> 
> as it is at 2.11 i believe this will be at the next self-hosted release which would be 1.4 if i'm not mistaken? cc: @ifbyol

@rberrelleza @teresaromero The flag `--all` in the destroy command is available since the `2.11.0` as Tere mentioned.

The changes affecting to `destroy namespace` and `destroy preview` operations will be avilable on `1.4`. I guess that part of the PR should be under `1.4` version and should be merged when the `1.4` is out

File: src/content/reference/cli.mdx [hunk 1]
```diff
@@ -249,6 +249,7 @@ $ okteto destroy
 | _--volumes_                              |  (bool)  | Destroy the persistent volumes created by the development environment (defaults to `false`)                                                             |
 | _--wait_                                 |  (bool)  | Wait until the development environment is destroyed (defaults to `false`)                                                                               |
 | _--no-bash_                              |  (bool)  | Execute the command using the container's default shell instead of bash (defaults to false)                                                             |
+| _--all_                                  |  (bool)  | Destroy everything at the current namespace (defaults to `false`)                                                                                       |
```
Comment (rberrelleza) [inline src/content/reference/cli.mdx:252]:
```suggestion
| _--all_                                  |  (bool)  | Destroy everything (defaults to `false`)                                                                                       |
```

File: src/content/reference/cli.mdx [hunk 2]
```diff
@@ -397,7 +398,7 @@ $ okteto namespace create test-cindy
 
 #### delete
 
-Delete a namespace.
+Delete a namespace. When `destroy` commands are defined at the development environments of the namespace, this will be executed.
```
Comment (rberrelleza) [inline src/content/reference/cli.mdx:401]:
```suggestion
Delete a namespace.  If the manifest includes `destroy` commands, they will be executed as part of the command.
```

File: src/content/reference/cli.mdx [hunk 3]
```diff
@@ -539,6 +540,7 @@ Run `okteto preview destroy` to destroy a given preview environment.
 
 You need to be [logged in](/docs/reference/cli/#context) to Okteto before running this command.
 Only users with administration privileges can destroy a global preview environment.
+When `destroy` commands are defined at the development environment of the preview, this will be executed.
```
Comment (rberrelleza) [inline src/content/reference/cli.mdx:543]:
```suggestion
 If the manifest includes `destroy` commands, they will be executed as part of this command.
```

**Review by rberrelleza — APPROVED** [review]

**Review by rberrelleza — APPROVED** [review]

**Pull request merged**